F5-Frontend-Enhancements-beta.js

```diff
@@ -1,13 +1,13 @@
 // ==UserScript==
```

Comment:
Why delete this one? There might be people running v11 still.

Reply:
Hmm. I guess I should look farther. What stops the current version from working on v11? I no longer have one to test against. I guess I can undelete that file.